features/networking/egress-ingress.feature

```diff
@@ -269,4 +269,42 @@ Feature: Egress-ingress related networking scenarios
       | curl | --head | www.baidu.com |
     Then the step should succeed
 
+  # @author huirwang@redhat.com
+  # @case_id OCP-13506
+  @admin
+  Scenario: Update different dnsname in same egress network policy
+    Given I have a project
+    Given I have a pod-for-ping in the project
+
+    # Create egressnetworkpolicy to deny www.test.com
+    Given I switch to cluster admin pseudo user
+    And I use the "<%= project.name %>" project
+    When I run oc create over "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/dns-egresspolicy/dns-egresspolicy4.json" replacing paths:
```

Comment:
Personal repository is used. We should add the file into v3-testfiles and use the files there.

Reply:
And we created EgressNetworkPolicy which might affect other scenarios, need to tag it as destructive to run it sequentially.

Reply:
@liangxia Thanks for the quick comments.  Updated the file to  in v3-testfiles and used the new link.
Actually EgressNetworkPolicy only take effect in project which it was created in, not affect other traffics. So it is okay to run in parallel。